packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +315,65 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// A default selection overlay for [CupertinoPicker]s.
+///
+/// It draws a gray rounded rectangle to match the picker visuals introduced in iOS 14
```

Comment:
This is a widget with a very specific use case so I think it could use a "see also":

This widget is typically only used in [CupertinoPicker.selectionOverlay]. In an iOS 14 multi-column picker, the selection overlay is a single rounded rectangle that spans the entire multi-column picker. To achieve the same effect using [CupertinoPickerDefaultSelectionOverlay], the additional margin and corner radii on the left or the right side can be disabled by turning off [capLeftEdge] and [capRightEdge], so this selection overlay visually connects with selection overlays of adjoining [CupertinoPicker]s (i.e., other "column"s).

See also:

* [CupertinoPicker], which uses this widget as its default [CupertinoPicker.selectionOverlay].